sample.env

```diff
@@ -91,6 +91,12 @@ LDAP_FILTER=
 #
 ALLOW_GREENLIGHT_ACCOUNTS=true
 
+# Last part of the Email-Address required for signup for a greenlight account
+#    domain.com matches also mail.domain.com
+#    @domain.com does NOT match @mail.domain.com
+#    multiple domains can be separated by whitespace and/or comma
+#REQUIRE_MAIL_DOMAIN=@domain.com, subdomain-allowed.net
```

Comment:
2 things: 
- Just for the sake of consistency, I'd prefer to name the ENV variable `LOCAL_ACCOUNT_HD` (or something along that sort)
- I would make it only support comma-separated strings `domain.com,subdomain-allowed.net`, since it would make everything else a lot cleaner